src/common/common.go

```diff
@@ -215,11 +216,10 @@ func HostMatchByNameOrIPAddress(node v1.Node, namesMap, IPAddressMap map[string]
 // Returns True when uninitialized taint must be removed.
 // This is required for some external platforms (e.g. VSphere, Nutanix) to proceed
 // with the installation using fake credentials.
-func RemoveUninitializedTaint(platform *models.Platform, invoker string, hasValidCredentials bool, openshiftVersion string) bool {
-	removeUninitializedTaintForPlatforms := [...]models.PlatformType{models.PlatformTypeNutanix, models.PlatformTypeVsphere}
+func RemoveUninitializedTaint(platformType string, invoker string, hasValidCredentials bool, openshiftVersion string) bool {
```

Comment:
why to change platformType type?

Reply:
In the case when assisted-service is unable, the platform type is determined by inspecting the install-config. That comes in as a string. So that is why RemoveUnitializedTaint looks at the string value.

Reply:
Not easier was to cast string to models.Platform at one place rather than changing previous code?

Reply:
I'll move it back to type. Not sure how to cast to models.Platform. Could you show me how?